src/XRControllerModel.ts

```diff
@@ -106,14 +118,24 @@ export class XRControllerModel extends Group {
     this.scene = null
   }
 
-  setEnvironmentMap(envMap: Texture, envMapIntensity = 1): XRControllerModel {
-    if (this.envMap === envMap && this.envMapIntensity === envMapIntensity) {
+  setEnvironmentMap(envMap: Texture | null): XRControllerModel {
+    if (this.envMap === envMap) {
```

Comment:
Isn't this a public method? Why split up the signature?

Reply:
It is a public method, but XRControllerModel wasn't really exposed up until https://github.com/pmndrs/react-xr/pull/294. Now that you've mentioned it, I see it could be done in a backwards compatible way

Splitting is needed for the following usecase:
You can an env map on a whole scene and materials that support it will just inherit them without any manually setting env map on materials inside controller model gltf scene. But there's a catch: controller models of a different devices sometimes have a completely different materials in terms of shininess and/or emissive and would either look too bright or too dark (depending on an env map). For that we tweak not the map itself, but the intensity on a controller materials, depending on a profile. And for that I would refrain from also updating the env map on the materials since it will unnecessarily complicate stuff.

For that reason I split it, but now I can see that we could keep the second optional argument on a setEnvironmentMap